### PR TITLE
RTL support

### LINKED ISSUE #35114: [Story interactive] Add RTL support for sliders
### Description

Sliders need to be RTL compliant by filling the correct side of the thumb, having the bubble in the correct place, alternate the size increase of the bubble depending on the direction, and have the min and max on the correct size.

### PR DESCRIPTION
**Pull request by syydneyyhardister**
Adds RTL support to the sliders, by correcting the placement of the thumb, bubble, and gradient.

Demo: https://sydney-story-demo.web.app/examples/amp-story/interactive_slider_rtl.html

Closes #35114

### CODE REVIEW

File: extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css [hunk 1]
```diff
@@ -46,6 +48,15 @@
   z-index: 1 !important;
 }
 
+[dir='rtl'] .i-amphtml-story-interactive-slider-container {
```
Comment (mszylkowski) [inline extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css:51]: This line should be `[dir='rtl'].i-amphtml-story-interactive-slider-container` (no space between them)

File: extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css [hunk 2]
```diff
@@ -20,6 +20,8 @@
   /* This maps the range and has the gradient follow behind the thumb */
   --percentage-corrected: calc(2.5% + 0.95 * var(--percentage)) !important;
   --thumb-size: 1.5em !important;
+  --gradient-angle: 90deg;
+  --translate-x-center: -50%;
```
Comment (mszylkowski) [inline extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css:24]: These can be `!important`

File: extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css [hunk 3]
```diff
@@ -55,10 +66,14 @@
...
   --scale-var: calc(var(--fraction) + 1) !important;
   /* Move the bubble up by sqrt(2)/2=70.7% to account for the pointy corner */
-  transform: translate(-50%, calc(-160% - var(--fraction) * 70.7%))
+  transform: translate(var(--translate-x-center), calc(-160% - var(--fraction) * 70.7%))
     scale(var(--scale-var)) !important;
 }
 
+/*[dir='rtl'].i-amphtml-story-interactive-mid-selection .i-amphtml-story-interactive-slider-bubble {
+  --translate-x-center: 50%;
+}*/
```
Comment (mszylkowski) [inline extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css:75]: If the other review comments make it work, we can remove these commented rules and merge it

**Review by mszylkowski — APPROVED** [review]

**Review by processprocess — APPROVED** [review]
💪 CSS variables

**Pull request merged**